index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+    <title>Priscilla Portfolio</title>
+</head>
+
+<body>
+    <div class="central-wrapper">
+        <header class="header-class">
+           <a href="#" target="_blank" rel="noopener noreferrer"><h3 class="logo-text">My Logo</h3></a>
+            <div class="hamburger">
+              <a href="#"><img src="resources/img/Union.png" alt=""></a> 
+            </div>
+        </header>
+        <main class="main-wrapper">
```

Comment:
You did great following best practices with this semantic tag `<main>`, but for the requirements of this project you should use a `<section>` tag :wink: